lib/vdsm/storage/exception.py

```diff
@@ -1622,10 +1622,21 @@ def __init__(self, lvname, tag):
     msg = "Missing logical volume tag."
 
 
-class LogicalVolumeDoesNotExistError(StorageException):
+class LogicalVolumeDoesNotExistError(VolumeLVMCommandError):
     code = 610
     msg = "Logical volume does not exist"
 
+    def __init__(self, vg_name, lv_name, error=None):
+        super().__init__(error)
+
+        self.vg_name = vg_name
+        self.lv_name = lv_name
+        self.error = error
```

Comment:
If we init error in the superclass, why do we override it here?

Reply:
agh, forgot to remove it (comes from another commit that had it).